### ISSUE TITLE
False positive - multi line strings

### ISSUE THREAD

**Issue by jamiehunt** [dral3x/StringsLint#19]
Receiving false positive violations:
```Missing Violation: Localized string "my_string" is missing (missing)```

For multi-lined strings, e.g.:

**Localizable.strings**
```
"my_string" = "Multi line strings are great...
This text is on line two.";
```

**Comment by dral3x** [comment]
Hi again @jamiehunt !

In my personal experience, multi-line localized strings are made as one-line with `\n` inside. 
I didn't know you could have them like in your example!

StringsLint parses files line by line, so having a multi-line string will simply break all parsers. 
Supporting them will require a big refactoring. 
Feel free to try that yourself and let me know if you need help deep diving into the code!
